installers/linux/al2/spec/java-amazon-corretto.spec.template

```diff
@@ -27,13 +27,18 @@
 %global release_id      $release_id
 %global version_opt     $version_opt
 %global experimental_feature     $experimental_feature
+%global addtional_configure_options     $addtional_configure_options
```

Comment:
spelling. should be 
additional_configure_options

Reply:
Thanks! good catch.